### PR TITLE
build(components-native): JOB-75931 - Update Components Native instructions to fix tests

### PR DESCRIPTION
**Pull request by MichaelParadis**
### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- Fixed `@jobber/components-native` to no longer break tests when installed with some of our dependencies

### Security

- 

## Testing

 - See Draft PR in Jobber Mobile to see the changes in action
 - https://github.com/GetJobber/jobber-mobile/pull/7775/files

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/index.ts [hunk 1]
```diff
@@ -42,3 +42,4 @@ export * from "./Switch";
 export * from "./Text";
 export * from "./Toast";
 export * from "./Typography";
+// export * from "./hooks";
```
Comment (MichaelParadis) [inline packages/components-native/src/index.ts:45]: To do: Remove this

**Review by taylorvnoj — COMMENTED** [review]
Looks good - I tested a few things and everything seems to work well.

Just for clarity - once we merge this and `components-native` gets installed back into mobile, we should actually look to add the MockForm.tsx, etc right? Should we have a ticket for that?

**Comment by MichaelParadis** [comment]
@taylorvnoj great point. I think we should add it as an AC / Subtask for the actual deprecation ticket. I can add it

**Review by taylorvnoj — APPROVED** [review]
Thank you for adding that to the future ticket! LGTM

**Pull request merged**